.github/workflows/check-formatting.yaml

```diff
@@ -6,7 +6,17 @@
 
 name: "Check formatting"
 
-on: [push, pull_request]
+on:
+  push:
+    paths:
+      - 'src/**'
+      - 'tests/**'
+      - 'libs/**'
+  pull_request:
+    paths:
+      - 'src/**'
+      - 'tests/**'
+      - 'libs/**'
```

Comment:
We probably want to also include:
- .github/workflows
- CMakeLists.txt

Check formatting also checks test source files.
So we probably also need:
- tests/
- **/*.erl

Reply:
Yes, we need `CMakeLists.txt` and `.github/workflows/**`, tests is already there, and I changed `libs/**` to `**/*.erl`